src/client/components/app/Transactions/DeployLineTx.tsx

```diff
@@ -155,7 +157,7 @@ export const DeployLineTx: FC<DeployLineProps> = (props) => {
         LinesActions.deploySecuredLineWithConfig({
           factory: LineFactory_GOERLI,
           borrower,
-          ttl: BigNumber.from(timeToLive),
+          ttl: BigNumber.from(ttl.toFixed(0)),
```

Comment:
this achieves the same thing as `Math.ceil()` do we care?

Reply:
This answers your question:

https://stackoverflow.com/questions/566564/math-roundnum-vs-num-tofixed0-and-browser-inconsistencies

I will make the necessary changes.

Reply:
I looked into this and BigNumber.from()  takes a string as an arguement, toFixed() returns a string, Math.ciel() returns a number.

https://docs.ethers.io/v5/api/utils/bignumber/
https://www.programmersought.com/article/2060735458/

let me know what you think